strings/ctype-czech.c

```diff
@@ -42,8 +42,8 @@
 	passes, that's why we need four times more space for expanded
 	string.
 
-	This file also contains the ISO-Latin-2 definitions of
-	characters.
+	The non-ASCII literal strings in this file are encoded in the iso-8859-2 / latin-2 character set
+  (https://en.wikipedia.org/wiki/ISO/IEC_8859-2)
```

Comment:
Another really minor thing, please align this with the rest of the comment. I think tabs were used for the rest of this comment block and spaces here.